docs/current_docs/api/snippets/clients-sdk/python/test.py

```diff
@@ -0,0 +1,46 @@
+"""Run tests for multiple Python versions concurrently."""
+
+import sys
+
+import anyio
+
+import dagger
+
+
+async def test():
+    versions = ["3.8", "3.9", "3.10", "3.11"]
+
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
```

Comment:
For custom apps, in all SDKs, please use the global client. See:
- https://github.com/dagger/dagger/issues/6296

With that change, this PR actually closes that issue.

Reply:
I've updated the Go example.

Python fails when doing `uv add`, using Python 3.11.2

```
(.venv) admin@ip-172-31-41-139:~/fastapi$ uv add dagger-io
  × No solution found when resolving dependencies:
  ╰─▶ Because dagger-io==0.1.0 was yanked (reason: Mistake creating tag in
      forked repo) and only dagger-io>=0.1.0 is available, we can conclude that
      dagger-io<0.1.1 cannot be used. (1)

      Because the requested Python version (>=3.7) does not satisfy Python>=3.10
      and dagger-io>=0.1.1,<=0.6.2 depends on Python>=3.10,<4.0, we can conclude
      that dagger-io>=0.1.1,<=0.6.2 cannot be used.
      And because we know from (1) that dagger-io<0.1.1 cannot be used, we can
      conclude that dagger-io<0.6.3 cannot be used. (2)

      Because the requested Python version (>=3.7) does not satisfy Python>=3.10
      and dagger-io>=0.6.3 depends on Python>=3.10, we can conclude that
      dagger-io>=0.6.3 cannot be used.
      And because we know from (2) that dagger-io<0.6.3 cannot be used, we can
      conclude that all versions of dagger-io cannot be used.
      And because your project depends on dagger-io and your project requires
      fastapi[all], we can conclude that your projects's requirements are
      unsatisfiable.

      hint: Pre-releases are available for dagger-io in the requested
      range (e.g., 0.1.0a1), but pre-releases weren't enabled (try:
      `--prerelease=allow`)

      hint: The `requires-python` value (>=3.7) includes Python versions that
      are not supported by your dependencies (e.g., dagger-io>=0.1.1,<=0.6.2 only
      supports >=3.10, <4.0). Consider using a more restrictive `requires-python`
      value (like >=3.10, <4.0).
  help: If you want to add the package regardless of the failed resolution,
        provide the `--frozen` flag to skip locking and syncing.
```

Reply:
TypeScript fails with this error

```
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL }
✔ connect 1.7s

Setup tracing at https://dagger.cloud/traces/setup. To hide: export GOAWAY=1

(node:2814) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2814) [DEP0180] DeprecationWarning: fs.Stats constructor is deprecated.
(Use `node --trace-deprecation ...` to show where the warning was created)

node:internal/modules/run_main:123
    triggerUncaughtException(
    ^
[Object: null prototype] {
  [Symbol(nodejs.util.inspect.custom)]: [Function: [nodejs.util.inspect.custom]]
}

Node.js v22.9.0
```

Reply:
@TomChv @helderco can you please review the error and the updated code, it's possible I made some errors when updating it?

Reply:
You're getting an error in uv because your project seems to require Python 3.7+, while the dagger library only supports Python 3.10+. 

The solution here can be just to put the Dagger script in a separate sub-project (like modules).

Or you can run it in an [isolated environment](https://docs.astral.sh/uv/guides/scripts/#running-a-script-with-dependencies):

```
dagger run uv run --no-project --with dagger-io test.py
```

Or you can also [put the dependency in the script](https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies) itself:
```
uv add --script test.py dagger-io
dagger run uv run test.py
```

Reply:
Python code works now, thanks @helderco !